### DISCUSSION TITLE
Problem installing Carla/ Unreal

### DISCUSSION THREAD

**Discussion by flschnepf** [carla-simulator/carla#7558] [Q&A]
Hello, 
I am relatively new to carla. I have already installed carla on another computer without any problems, but now I got a problem.
I want to install carla on ubuntu 20.04.06. However, I get an error message when I start the unreal editor.

![Screenshot from 2024-05-03 09-21-30](https://github.com/carla-simulator/carla/assets/158583663/b3a03989-1583-467c-ba86-e0fb1c8cbe6d)

According to my research, the problem should be solved with the latest nvidia driver. With version 535.171.04 I should have the latest one for the rtx 2080Ti, but I still have the problem. What do I have to do?

I would be very thankful if someone could help me

**Comment by evoevo07** [comment]
Not really a solution, but my experience:
I've constantly had/have issues with Vulkan and Nvidia drivers.
It worked for me after I tried separately whether the drivers, Vulkan, and so on were working correctly, aside from Carla.
In my case, it was even the case that the hardware(A1000 Laptop) is too new for the kernels in Ubuntu 20.04, so I have to use 22.04.
A lot of headaches...

**Comment by evoevo07** [comment]
There are some test functions for Vulkan.
(Vulkaninfo, and a command which displays a spinning cube (forgot the name of the command))

**Reply by flschnepf** [reply]
Thanks for sharing your experience. 
I think you mean vkcube. It recognizes my gpu correctly. Then I tested my gpu with glmrk2 and got a score of 11355.
Do you know what else I can test?

**Reply by evoevo07** [reply]
Yeah, it's vkcube.
No sorry, I don't have any Ideas anymore.
Maybe try another driver.
